lib/Utilities.php

```diff
@@ -0,0 +1,89 @@
+<?php
+class sspmod_expirychecker_Utilities {
+
+  /**
+  * Expects three strings for a url and what marks out the beginning
+  * and end of the domain. 
+  * 
+  * Returns a string with the domain portion of the url (e.g. www.insitehome.org)
+  */
+    public static function getUrlDomain($in_url, $start_marker='//', 
```

Comment:
Probably not... that `getUrlDomain()` was leftover/inherited code. I've now added issue #2 for this... thanks for pointing it out.